src/include/table.h

```diff
@@ -43,6 +43,8 @@ namespace emerald {
 
```

Comment:
I think this file isn't needed too, but I haven't removed it yet so just leave it first.